### ISSUE TITLE
Use stock Twitch/FFZ chat font and button colors?

### ISSUE THREAD

**Issue by kevronicoal** [night/BetterTTV#1859]
I recently added BTTV to Chrome, and enjoy some of its features. However, I've grown very used to using FFZ with Twitch, and one feature that disrupts the feel of Twitch that I've enjoyed is the fonts that BTTV adds on to the site.

Basically, BTTV makes the fonts in chat bold or larger. I like the normal Twitch font (or, at least what FFZ uses, unless FFZ does not modify the fonts at all), which are non-bold. It would be cool to have a setting to disable any font modification that BTTV does to Twitch.

Also, I like the colors that I usually have for my emoticon button (the smiley face), the bits button, and my FFZ button (all in the chat bar). When BTTV is enabled, these colors are brighter gray, and look a little larger/blurry (and darker red for the FFZ button, depending on channel). I like the colors and looks of the whole chat bar that I see when I use just FFZ.

tl;dr: I like the look of chat and the buttons when I only use FFZ. Adding on BTTV makes fonts look bold, and buttons/lines along the chat border look brighter and slightly bolder. There should be a way to disable any of the visual modifications like these that BTTV adds. In a way, I would like to use BTTV solely for its emotes and a couple other features. Visually, it has made me keep it disabled for now.

**Comment by thinkpad4** [comment]
FailFish

**Comment by night** [comment]
While we appreciate your feedback, BetterTTV is primarily what we think makes a better experience. Having configuration to control every aspect of a product is something people assume to be possible, but in reality it is a headache to maintain, a burden for newcomers to learn, and is just overall confusing behavior for users (literally it's the epitome of a phenomenon called "feature creep"). We're happy you quite enjoy the 100+ options FFZ provides, but we're providing a more streamlined approach for the general public here, sorry.
